doc/md/examples/grammar.txt

```diff
@@ -214,6 +214,7 @@
     'break' <id> <exp_nullary>?
     'continue' <id>
     'debug' <exp_nest>
+    '(' <exp_post>? 'with' <list(<exp_field>, ';')> ')' <exp_nest>
```

Comment:
Thoughts on using `,` here in place of `;` for consistency with other parenthesized expressions? The formatter uses this invariant (commas in parentheses, semicolons in square brackets and curly braces) for automatically replacing commas with semicolons and vice versa whenever there is otherwise a syntax error that makes the AST unparseable. I can add an exception, but it seems nice to keep this pattern so that it's easier for people to remember which delimiter to use.

Reply:
Sorry for missing this comment for some time... Yeah, it is a nice consistency argument. I was stealing the syntax from the record field separators, but I guess comma works as well. Will try and report back. @crusso any gut feelings about this?

Reply:
I'd stick with `;` since these are more like fields. Would be even nicer if they just were fields...

Reply:
Yes, they are fields, but where is it written that they cannot be separated by commas? ;-) Just playing the devil's advocate.

Anyway, I have started a branch to get a feeling for the suggestion: #4782. I am not married to either way.